changelog/fragments/1669234779-Fix-upgrade-under-V2.yaml

```diff
@@ -0,0 +1,31 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: bug-fix
+
+# Change summary; a 80ish characters long description of the change.
+summary: Fix upgrade under V2
+
+# Long description; in case the summary is not enough to describe the change
+# this field accommodate a description without length limits.
+#description:
+
+# Affected component; a word indicating the component this changeset affects.
+component: Fix a typo that was preventing upgrades under V2.
```

Comment:
This isn't a released bug so we probably don't need a user facing changelog entry for it, even though the linter complains about it anyway.,

Reply:
I agree, adding the changelog just makes noise. The CI check is not mandatory, thus I guess it's fine having it fail when we are sure there should not be a changelog

Reply:
Indeed, I made it mostly out of habit. I'll remove it.